app/src/main/java/org/wikipedia/talk/TalkReplyActivity.kt

```diff
@@ -462,14 +479,25 @@ class TalkReplyActivity : BaseActivity(), LinkPreviewDialog.Callback, UserMentio
 
     override fun onBackPressed() {
         setResult(RESULT_BACK_FROM_TOPIC)
+        if (viewModel.isFromDiff) {
```

Comment:
We can remove this `if` statement.

Reply:
I removed the other duplicate since we want this on back press.